kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/PropertyObserver.kt

```diff
@@ -19,17 +19,19 @@
 
 package org.eclipse.kuksa
 
+import org.eclipse.kuksa.pattern.listener.Listener
 import org.eclipse.kuksa.proto.v1.Types.DataEntry
+import org.eclipse.kuksa.proto.v1.Types.Field
 import org.eclipse.kuksa.vsscore.model.VssSpecification
 
 /**
  * The Observer is used to notify about changes to subscribed properties.
  */
-interface PropertyObserver {
+interface PropertyObserver : Listener {
```

Comment:
Observing or listening - Maybe we should decide on one terminology.

Reply:
Renamed this as well as the other Observer classes to PropertyListener, SpecificationPropertyListener, VssSpecificationListener